### DISCUSSION TITLE
GPG signing of commits

### DISCUSSION THREAD

**Discussion by tarkatronic** [underdog-tech/vulnbot#41] [Polls]
Right now, as with all of our company's code, this project requires that all commits be signed by the author. Which is great from a security and identity verification perspective. As I've thought about it though, this is a pretty high bar for contributors to an Open Source project. And I want to do everything I can to _encourage_ new contributors, not make it more difficult for them.

So this brings me to my question...

**Comment by JoseAngel1196** [comment]
I think either way works fine. If we require GPG, then we should have documentation on how to set up GPG commits for new contributors in our README.md.
